js/components/discussions/thread.vue

```diff
@@ -133,6 +104,18 @@ export default {
             return moment(this.discussion.closed).format('LL');
         }
     },
+    ready: function() {
+        if( this.discussion.closed_by ){
+            const user_id = this.discussion.closed_by;
+
+            this.detailed = false;
+
+            this.$api.get('users/${user_id}').then(response => {
+                this.closed_by_url = response.page;
+                this.closed_by_name = response.first_name + " " + response.last_name;
```

Comment:
this.closed_by_name = `${response.first_name} ${response.last_name}`;
But if `closed_by` always have to be resolved, maybe it's better to properly serialize the user [here](https://github.com/opendatateam/udata/blob/master/udata/core/discussions/api.py#L41-L43) to avoid the extra queries (it's [already done for the author](https://github.com/opendatateam/udata/blob/master/udata/core/discussions/api.py#L37-L38):